test/token/ERC721/ERC721BasicToken.behavior.js

```diff
@@ -221,6 +221,12 @@ function shouldBehaveLikeERC721BasicToken (accounts) {
           });
         });
 
+        context('when the address of the previous owner is null', function () {
```

Comment:
That `require` only causes us to revert early anyway, since functions are later called with an equivalent `require`. I'd also remove it in that case, since we're already testing for unauthorized users.